### DISCUSSION TITLE
Welcome to CARACal Discussions!

### DISCUSSION THREAD

**Discussion by o-smirnov** [caracal-pipeline/caracal#1296] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KshitijT** [comment]
Hi all, I am Kshitij. I work as a postdoctoral fellow at the University of Pretoria (courtesy IDIA). 
I (sometimes) dabble in writing very astrophysic-y sort of code (read: very brute-force and quite devoid of comments) for our beloved pipeline, CARACal.

**Comment by gigjozsa** [comment]
Josh: Dr. Gyula I. G. Józsa, Senior Astronomer, SARAO, Visiting Professor, Department of Physics and Electronics, Rhodes University
Prime user of CARACal and I also spent a bit of time contributing to it. "The pipeline! Paolo, the pipeline!"

**Comment by PeterKamphuis** [comment]
Hi all, I am Dr. Peter Kamphuis and am currently employed within the D-MeerKAT project in Germany at the Ruhr University. I have used Caracal for reducing GMRT GSB data and contributed some code to it (Mostly the selfcal module). Currently I am in the process of setting up a HPC cluster in Germany where CARACal can be deployed for reducing open time MeerKAT data and further development.

**Comment by o-smirnov** [comment]
Hehe, I didn't expect everyone to take this so seriously and formally! I am Oleg and I just click buttons here... The Welcome text is github boilerplate which resulted from my button clicking -- I hope you didn't *really* think I would write something so cheesy? **cringe** :rofl: 

Anyway, this new feature looks like a useful alternative to the more formal issue tracker, i.e. for the more informal and open-ended issues that we've been labelling as "Help wanted" and "Question" and "Mirto" and stuff like that. I see there are buttons for moving Issues to Discussions and vice versa, so let's just see how it works...

**Comment by KshitijT** [comment]
Yes, I have something to discuss, where are those Mirtos? ;) 
Last I counted, should've got 1 mirto, 1 amarula and 2 and a half of those medals @gigjozsa was talking about.... :P
